src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/context.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'context()'
+excerpt: 'Browser module: context method'
+---
+
+Returns the current [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/).
+
+<Blockquote mod="note" title="">
+
+A 1-to-1 mapping between [Browser](/javascript-api/k6-experimental/browser) and `BrowserContext` means you cannot run `BrowserContexts` concurrently. If you wish to create a new `BrowserContext` while one already exists, you will need to [close](/javascript-api/k6-experimental/browser/browsercontext/close) the current one, and create a new one with either [newContext](/javascript-api/k6-experimental/browser/newcontext/) or [newPage](/javascript-api/k6-experimental/browser/newpage). All resources associated to the closed `BrowserContext` will also be closed and cleaned up (such as [Page](/javascript-api/k6-experimental/browser/page/)s).
+
+</Blockquote>
+
+### Returns
+
+| Type           | Description                                                                                                                              |
+| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| object \| null | The current [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/) if one has been created, otherwise `null`. |
+
+
+### Example
+
+```javascript
+import { browser } from 'k6/experimental/browser';
+
+export const options = {
+  scenarios: {
+    browser: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+            type: 'chromium',
+        },
+      },
+    },
+  },
+}
+
+export default function () {
+  console.log(browser.context()); // null
+
+  const page = browser.newPage();
+  const context = browser.context();
+  console.log(context); // {"base_event_emitter":{}}
```

Comment:
I'm not sure we should reference the `base_event_emitter` in JS API docs example, as it seems a pretty internal element, even thought that is what would actually be printed, but I think It might confuse users. What do you think?

Would it work an example such as:
```suggestion
  console.log(browser.context()); // null

  const page1 = browser.newPage(); // implicitly creates a new browserContext
  const context = browser.context(); // underlying live browserContext associated with browser
  const page2 = context.newPage(); // shares the browserContext with page1
```
Maybe too many comments though.

Reply:
Yeah, i was wondering the same. I like your suggestion 👍